src/sentry/models/project.py

```diff
@@ -348,6 +348,7 @@ def transfer_to(self, team):
         except IntegrityError:
             slugify_instance(self, self.name, organization=organization)
```

Comment:
It's too bad this doesn't return the slug, I'd think it'd be clearer and less error-prone to make this part of the `update` call below. 😞